yjit.rb

```diff
@@ -390,6 +390,12 @@ def _print_stats(out: $stderr) # :nodoc:
       out.puts "yjit_alloc_size:       " + format_number(13, stats[:yjit_alloc_size]) if stats.key?(:yjit_alloc_size)
       out.puts "live_context_size:     " + format_number(13, stats[:live_context_size])
       out.puts "live_context_count:    " + format_number(13, stats[:live_context_count])
+
+      bytes_per_context = stats[:context_data_bytes].fdiv(stats[:num_contexts_encoded])
+      out.puts "context_data_bytes:    " + format_number(13, stats[:context_data_bytes])
+      out.puts "num_contexts_encoded:  " + format_number(13, stats[:num_contexts_encoded])
```

Comment:
Do we need both `live_context_size`/`live_context_count` and `context_data_bytes`/`num_contexts_encoded`?

Reply:
No we don't. The live_context_size / live_context_count are the old stats. I kept them around for comparison purposes but I can get rid of them since they will no longer be relevant.